samples/openapi3/client/petstore/python/tests/test_pet_api.py

```diff
@@ -171,18 +171,17 @@ def test_add_pet_and_get_pet_by_id(self):
         self.assertIsNotNone(fetched.category)
         self.assertEqual(self.pet.category.name, fetched.category.name)
 
-    def test_add_pet_and_get_pet_by_id_with_preload_content_flag(self):
-        try:
-            self.pet_api.add_pet(self.pet, _preload_content=False)
-        except ValueError as e:
-            self.assertEqual("Error! Please call the add_pet_with_http_info method with `_preload_content` instead and obtain raw data from ApiResponse.raw_data", str(e))
+    def test_add_pet_and_get_pet_by_id_without_preload_content_flag(self):
 
         self.pet_api.add_pet(self.pet)
-        fetched = self.pet_api.get_pet_by_id_with_http_info(pet_id=self.pet.id,
-                                                            _preload_content=False)
-        self.assertIsNone(fetched.data)
-        self.assertIsInstance(fetched.raw_data, bytes) # it's a str, not Pet
-        self.assertTrue(fetched.raw_data.decode("utf-8").startswith('{"id":'))
+        fetched = self.pet_api.get_pet_by_id_without_preload_content(pet_id=self.pet.id)
+        read = fetched.read()
+        self.assertIsInstance(fetched, urllib3.HTTPResponse)
```

Comment:
A test on the added `_without_preload_content` can be found here.
`_without_preload_content` returns `urllib3.HTTPResponse`.

Reply:
You could add a similar test for the aiohttp client, the one from https://github.com/OpenAPITools/openapi-generator/pull/16789 should almost work.